engine/gui/src/test/test_gui_script.cpp

```diff
@@ -49,12 +49,15 @@ static dmLuaDDF::LuaSource* LuaSourceFromStr(const char *str)
 }
 
 void GetTextMetricsCallback(const void* font, const char* text, float width, bool line_break, float leading, float tracking, dmGui::TextMetrics* out_metrics);
+void RenderNodesStoreTransform(dmGui::HScene scene, const dmGui::RenderEntry* nodes, const dmVMath::Matrix4* node_transforms, const float* node_opacities,
```

Comment:
I was a bit annoyed that we had two "RenderScene" functions in the gui.
We should use the one we use in the engine for the tests too.